libfreerdp/crypto/test/TestFuzzCryptoErReadContextualTag.c

```diff
@@ -0,0 +1,10 @@
+#include <freerdp/crypto/crypto.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	bool pc = true;
+	BYTE tag = "";
+	er_read_contextual_tag((wStream*)Data, tag, &Size, pc);
```

Comment:
this is really wrong, you can not simply cast any buffer to a `wStream` and expect it to not crash

Reply:
Done.